game.py

```diff
@@ -530,6 +686,36 @@ def _circle(self, r, cx, cy):
     def _footer(self):
         return '</svg>\n'
 
+    def save_highscore(self):
+        file_path = os.path.join(self._activity_root, 'data', 'highscore')
+        logging.debug(file_path)
+        highscore = []
+        highscore.append(0)
+        if os.path.exists(file_path):
+            File = open(file_path, "r")
+            highscore = File.readlines()
+            File.close()
```

Comment:
I suppose its better to use `with..as` block for file handling, to prevent leaving open file handles.

```suggestion
            with open(file_path, "r") as fp:
                highscore = fp.readlines()
```